CMakeLists.txt

```diff
@@ -323,14 +323,13 @@ else()
   add_subdirectory(3party/icu)
 endif()
 
-set(JANSSON_EXAMPLES OFF)
 set(JANSSON_BUILD_DOCS OFF)
-set(JANSSON_WITHOUT_TESTS ON)
+set(JANSSON_BUILD_MAN OFF)
+set(JANSSON_EXAMPLES OFF)
 set(JANSSON_INSTALL OFF)
+set(JANSSON_WITHOUT_TESTS ON)
 find_package_or_fallback_to_3party(Jansson 2.14.0 3party/jansson/jansson/)
-if (NOT Jansson_FOUND)
-  include_directories("${PROJECT_BINARY_DIR}/3party/jansson/jansson/include")
-endif()
+target_include_directories(jansson INTERFACE "${PROJECT_BINARY_DIR}/3party/jansson/jansson/include")
```

Comment:
So, as I understand, if we avoid system package (by replacing include dir), also better to replace
find_package_or_fallback_to_3party above with add_subdirectory(3party) to avoid any possible conflicts, no?

Reply:
I think we should simply have opt-in flags for system provided headers, and then those who don't need them could rest easy.